temporal/api/cloud/nexus/v1/message.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package temporal.api.cloud.nexus.v1;
+
+option go_package = "go.temporal.io/api/cloud/nexus/v1;nexus";
+
+import "google/protobuf/timestamp.proto";
+
+message IncomingServiceSpec {
+    // The name of the incoming service. Must be unique within an account
```

Comment:
Open Question:

For OSS, the "name" factors into the URL used when tying to access the service from a cluster.

For Cloud, the URL for addressing of services would be an internal implementation detail and generated by the server. In other words, we aren't using the incoming service name for anything here, so technically, we 1) do not need to enforce uniqueness and 2) this could be a friendly name/description instead. So should we still follow the same semantics here? We can do so just for consistency, but want to make sure there is "value" to the customer here.
3) Can we get away with making this immutable?

Reply:
conversely, we can require the outgoing service reference this name directly.